### DISCUSSION TITLE
Apple menu options?

### DISCUSSION THREAD

**Discussion by aleph1** [neutralinojs/neutralinojs#955] [Q&A]
I have successfully got an application up and running in Neutralino, but can't see any information in the documentation or issues regarding having items in the Apple menu for the application. I want to be able to use a standard keyboard control for quitting (⌘Q) and ideally have the Apple menu for the application have one item that reads “Quit AppName ⌘Q”. Is this possible?

**Comment by Blatman** [comment]
Unfortunately I'm not sure you will get a positive answer for this since Mac native menus seem to be in the too hard basket. The closest I saw was the PR#237 on the webview repo (upstream) which appeared to work but the core webview.h code changed significantly and considers mac menuing as non-core. Interestingly I noticed that there was an attempt to integrate the changes into Tauri. The other method I tried was using osascripting but unfortunately within the Neutralino webview I can only get it to interact with the tray part of the topbar so the usual menubar hook must be keyed to the systray instead (if I use the osascript method with say Finder the the normal menubar can be accessed). So I guess a C++ expert would have to start hacking the webview to get some native menus.

**Comment by aleph1** [comment]
Thanks. Yeah, I have this working as expected in a Tauri wrapper. I’m actually reasonably proficient in C++ but too busy to find time to look at what would need to change to make this work.
